src/components/Editor/Tabs.js

```diff
@@ -85,6 +85,11 @@ const SourceTabs = React.createClass({
     if (!(prevProps === this.props)) {
       this.updateHiddenSourceTabs();
     }
+    const { selectedSource, sourceTabs, selectSource } = this.props;
+    if (!selectedSource && sourceTabs.size > 0) {
+      const source = sourceTabs.get(0);
+      selectSource(source.get("id"));
+    }
```

Comment:
I just tested it locally and it looks like rendering without a selected source works.

Lets take this out and we can always figure out the best thing to do later.
The problem with this is that we're changing state. i.e. if the previously selected source is loaded soon after we do this we're wrong :)